### ISSUE TITLE
Universalload

### ISSUE THREAD

**Issue by Migrst** [loki79uk/FS25_UniversalAutoload#197]
How is it porssible to use the funktions on a Stema traius Trailer ( Tip and ramps ) when using universalload

And on the Krone trailer it not porssible to open the siden then using it

Can you help me ?

**Comment by loki79uk** [comment]
Do you mean this? https://github.com/loki79uk/FS25_UniversalAutoload/issues/86

I think you will need to change your keybindings so that there are no conflicts.

**Comment by Migrst** [comment]
The trailer uses Y and U for opening sides and back and ramps    and they work 
But dont show up when using F1 menu

**Comment by loki79uk** [comment]
What happens if you re-bind those keys from UAL to something else?  or unbind them completely?  It is only possible to display something like 8 actions in the F1 menu at one time.

**Comment by Migrst** [comment]
yes and whit the universalload you have 8 actions so there a no room for any other funktions

I have made the funktions Y and U      to Æ and Ø      but still dont show up because to many actions

**Comment by loki79uk** [comment]
I can't do anything about that - you have to choose what controls to bind.  Usually they will work even if not displayed on the F1 menu, but this os one part of the game that is pretty terrible.  I don't see why GIANTS couldn't allow us to have 20 different bindings visible.

**Comment by loki79uk** [comment]
> Could the Universal load just be simplified so you cant those what you want to load but just load it ? It who'd make room for other funktions

That is what I mean, you can unbind those UAL functions if you want to - no one is forcing you to use them.

**Comment by Migrst** [comment]
Ah ok i try

**Comment by Migrst** [comment]
Yes that Works      but will be really nice if more key bindings who'd be possible

But thanks

**Comment by loki79uk** [comment]
Closing as not a bug, but maybe submit a feature request to GIANTS to allow more keybindings in F1 menu.
